src/vs/workbench/browser/parts/editor/media/titlecontrol.css

```diff
@@ -26,6 +26,10 @@
 	cursor: pointer;
 }
 
+.monaco-workbench .part.editor > .content .editor-group-container > .title .monaco-icon-label::after {
+	padding-right: 0;
```

Comment:
I have done that to prevent that large gap from showing between decoration and commands. Tho, you screen shots look very different. Decoration appear right aligned. Is that some setting?

**tabs**
<img width="391" alt="Screenshot 2020-12-16 at 11 47 19" src="https://user-images.githubusercontent.com/1794099/102338718-81c7d980-3f94-11eb-849f-f75e60e9b756.png">

**no tabs**
<img width="588" alt="Screenshot 2020-12-16 at 11 47 56" src="https://user-images.githubusercontent.com/1794099/102338750-8be9d800-3f94-11eb-9649-67a35f226b66.png">

Reply:
@jrieken yeah check with breadcrumbs disabled, that is what I had running

Reply:
I see, so no tabs and no breadcrumbs... Wondering if that right alignment of decorations is actually wanted in that case or if they should stick to the label.

Reply:
Yeah good point, I am fine having it close to the beginning of the label to keep it consistent. But I was also not opposed as it is now, given how I think other editors I have used in the past typically showed errors/warnings all the way to the right.

![image](https://user-images.githubusercontent.com/900690/102341794-a625b500-3f98-11eb-86d6-ee57c694de7d.png)

Reply:
This is actually hugely and complicated because without breadcrumbs, decorations show after the folder label and with breadcrumbs they show after the name label. That's because breadcrumbs aren't part (a child) of the resource label